src/node/datomish/cljify.cljs

```diff
@@ -1,7 +1,27 @@
 (ns datomish.cljify)
 
 (defn cljify
-  "In node, equivalent to `(js->clj o :keywordize-keys true).
-  See <http://dev.clojure.org/jira/browse/CLJS-439?focusedCommentId=43909>."
-  [o]
-  (js->clj o :keywordize-keys true))
+  "In node, equivalent to `(js->clj o :keywordize-keys true),
+  but successfully passes Clojure Records through. This allows JS API
+  callers to round-trip values they receive from ClojureScript APIs."
+  [x]
+  (if (record? x)
```

Comment:
It's very similar, with the differences being no options and the ability to hook our record check in for recursing down into objects. I'll add a comment.